### PR TITLE
Cockpit 252

### PR DESCRIPTION
**Pull request by skobyda**
Preview: https://skobyda.github.io/cockpit-project.github.io/blog/cockpit-252.html

### CODE REVIEW

File: _posts/2021-09-01-cockpit-252.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Try it out
+
+Cockpit 252 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/252)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [Cockpit Fedora 33](https://bodhi.fedoraproject.org/updates/?releases=F33&packages=cockpit)
```
Comment (martinpitt) [inline _posts/2021-09-01-cockpit-252.md:43]: We dropped F33 and added F35 -- makes sense to update the template in the  generation script

File: _posts/2021-09-01-cockpit-252.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 252
+author: skobyda
+date: '2021-09-01'
+tags: cockpit
+slug: cockpit-252
+category: release
+summary: ''
```
Comment (martinpitt) [inline _posts/2021-09-01-cockpit-252.md:8]: Please add a summary

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: _posts/2021-09-01-cockpit-252.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## webserver: Drop remotectl utility
+
+[Cockpit 241](https://cockpit-project.org/blog/cockpit-241.html) already stopped using the deprecated `remotectl` utility for creating certificates for the main `cockpit.service`, in favor of the much simpler `/usr/libexec/cockpit-certificate-ensure`. This version moves the remaining usage (containers and documentation) to that helper, and drops `remotectl`.
+
+If you use scripts, playbooks, or similar direct invocations of `remotectl`, please consider removing that: Cockpit creates or refreshes a self-signed certificate on startup automatically if required. If you must pre-create it, please move your scripts to call
+`/usr/libexec/cockpit-certificate-ensure`. However, please note that this script is *not* a stable and supported public interface.
+
+## shell: Show package version in 'About web console' modal
```
Comment (martinpitt) [inline _posts/2021-09-01-cockpit-252.md:24]: Let's capitalize "shell" for consistency, and say version*s*. Please adjust in the PR and use the script to regenerate the blog, to keep these two  in sync for as long as possible/desirable.

**Comment by martinpitt** [comment]
@skobyda: All bodhi URLs for all 3 projects exist now.

File: _posts/2021-09-01-cockpit-252.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2021-09-01-cockpit-252.md:43]: Done in PR #452, will need to update this PR -- either manually, or by  re-running the script.

File: _posts/2021-09-01-cockpit-252.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/252)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/251)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-machines)
+* [cockpit-machines Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-machines)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/34)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
+* [cockpit-podman Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-podman)
```
Comment (martinpitt) [inline _posts/2021-09-01-cockpit-252.md:49]: Please fill in the actual bodhi URLs from https://bodhi.fedoraproject.org/updates/?packages=cockpit and -machines and -podman

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: _posts/2021-09-01-cockpit-252.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+If you use scripts, playbooks, or similar direct invocations of `remotectl`, please consider removing that: Cockpit creates or refreshes a self-signed certificate on startup automatically if required. If you must pre-create it, please move your scripts to call
+`/usr/libexec/cockpit-certificate-ensure`. However, please note that this script is *not* a stable and supported public interface.
+
+## Shell: Show package versions in 'About web console' modal
+
+![Screenshot from 2021-08-30 14-07-52about]({{ site.baseurl }}/images/252-shell-show-package-versions-in-about-web-console-modal.png)
+
+## Storage: Encryption is presented as a property of a Filesystem
+
+Cockpit used to make the encryption layer of a encrypted filesystem explicit: It had its own row in the content table and it would be referred to in other places in the UI.  Now the encryption layer is hidden away and Cockpit pretends encryption is a inherent property of a Filesystem.
```
Comment (KKoukiou) [inline _posts/2021-09-01-cockpit-252.md:30]: of a Filesystem? Why is that capitalized?

**Review by KKoukiou — CHANGES_REQUESTED** [review]

File: _posts/2021-09-01-cockpit-252.md (hunk 5 above)
Comment (KKoukiou) [reply _posts/2021-09-01-cockpit-252.md:30]: If that's conscious because it's the tab name I am fine with that.

Comment (skobyda) [reply _posts/2021-09-01-cockpit-252.md:30]: That was inhereted from here https://github.com/cockpit-project/cockpit/pull/13192#issue-345738158

Will change it

**Review by martinpitt — APPROVED** [review]
Dakujem! Looks good now.

**Pull request merged**